janusgraph-core/src/main/java/org/janusgraph/diskstorage/log/kcvs/KCVSLog.java

```diff
@@ -337,9 +337,23 @@ public synchronized void close() throws BackendException {
                 }
             }
         }
-        writeSetting(manager.senderId, MESSAGE_COUNTER_COLUMN, numMsgCounter.get());
-        store.close();
-        manager.closedLog(this);
+        try {
+            writeSetting(manager.senderId, MESSAGE_COUNTER_COLUMN, numMsgCounter.get());
+        } catch (Throwable e) {
+            log.error("Could not persist message counter [" + manager.senderId + "] ; message counter ["+numMsgCounter.get()+"]",e);
```

Comment:
We could avoid unnecessary string concatenation by using slf4j placeholders:
```suggestion
            log.error("Could not persist message counter [{}] ; message counter [{}]", manager.senderId, numMsgCounter.get(), e);
```